tests/console/yast2_rmt.pm

```diff
@@ -51,8 +52,11 @@ sub test_ui {
     assert_screen "yast2_rmt_ssl_CA_password";
     type_password_twice;
     assert_screen "yast2_rmt_firewall";
-    send_key "spc";
+    # skip open firewall ports for tumbleweed bsc#1207390
+    send_key "spc" unless is_tumbleweed;
     send_key "alt-n";
+    assert_screen "yast2_rmt_firewall_ports_not_open" if is_tumbleweed;
+    send_key "alt-i" if is_tumbleweed;
```

Comment:
would be much more readable and a classic `if/else` here instead of many inline `if,unless`.